src/windows.rs

```diff
@@ -393,3 +397,67 @@ pub fn lang() -> impl Iterator<Item = String> {
 
     LangIter { array, index }
 }
+
+/// The processor architecture of the installed operating system.
+// The variants listed below are only platforms where Rust toolchain is
+// supported, to see the full list of architectures supported by Windows, see
+// https://docs.rs/winsafe/latest/winsafe/co/struct.PROCESSOR_ARCHITECTURE.html
```

Comment:
I think it makes sense to include the full list here.  Is there a Microsoft winapi docs link we can use instead?

Reply:
The link for the doc from MS is [here](https://learn.microsoft.com/en-us/windows/win32/api/sysinfoapi/ns-sysinfoapi-system_info#members), but, as you can see, it is not a full list.

 I guess that the list given by MS is the arches that are currently actively maintained, other arches are no longer supported or rarely used.

Reply:
Ok, I guess the link should be kept as-is, then.